cores/esp8266/Updater.cpp

```diff
@@ -104,7 +104,9 @@ bool UpdaterClass::begin(size_t size, int command, int ledPin, uint8_t ledOn) {
   _reset();
   clearError(); //  _error = 0
 
+#ifndef HOST_MOCK
   wifi_set_sleep_type(NONE_SLEEP_T);
+#endif
```

Comment:
Can this be defined in the user_interface.cpp mock to avoid ifndef?

Reply:
It actually is defined there already, but unfortunately `user_interface.cpp` is not for CI testing but for building host apps and including that ends up eventually requiring the whole host app sources (with its own main, etc.).

The host app and host tests have differing build setups and assumptions (and goals, obviously).  It's probably possible to refactor them completely to avoid the duplication here, but I'm not so sure this is the right time for it (3.0.0-ish feels right).